flight/tests/dsm/Makefile

```diff
@@ -1,6 +1,6 @@
 ###############################################################################
 # @file       Makefile
-# @author     Tau Labs, http://taulabs.org, Copyright (C) 2014
+# @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2013
```

Comment:
??

Reply:
Haha.  Copied over Makefile from the flash test :P